server/public/model/link_metadata.go

```diff
@@ -21,6 +21,7 @@ const (
 	LinkMetadataTypeNone      LinkMetadataType = "none"
 	LinkMetadataTypeOpengraph LinkMetadataType = "opengraph"
 	LinkMetadataMaxImages     int              = 5
+	LinkMetadataMaxURLLength  int              = 2048
```

Comment:
Please add a comment saying that this is the max length of the database column `url` in `linkmetadata` table.

Reply:
https://github.com/mattermost/mattermost/pull/29589/commits/9a9d2609b73ce0a4ccc6519f909ed657ffefb484